sql/mob_pools.sql

```diff
@@ -3828,7 +3828,7 @@ INSERT INTO `mob_pools` VALUES (3772,'Stickpin','Stickpin',172,0x000014010000000
 INSERT INTO `mob_pools` VALUES (3773,'Stihi','Stihi',87,0x0000A40100000000000000000000000000000000,1,1,12,240,100,0,1,1,1,2,0,32,2108,155,0,0,0,0,0,87,87);
 INSERT INTO `mob_pools` VALUES (3774,'Stinging_Sophie','Stinging_Sophie',48,0x0000100100000000000000000000000000000000,1,1,2,240,100,0,0,0,1,2,0,32,0,669,8,0,0,0,0,48,48);
 INSERT INTO `mob_pools` VALUES (3775,'Stink_Bats','Stink_Bats',47,0x0000040100000000000000000000000000000000,1,1,11,240,100,0,0,0,1,0,0,64,166,641,8,0,0,0,0,47,47);
-INSERT INTO `mob_pools` VALUES (3776,'Stirge','Stirge',46,0x0000000100000000000000000000000000000000,1,1,11,240,100,0,1,0,1,0,0,0,397,643,8,0,0,0,0,46,46);
+INSERT INTO `mob_pools` VALUES (3776,'Stirge','Stirge',46,0x0000000100000000000000000000000000000000,1,1,11,240,100,0,0,0,1,0,0,0,397,643,8,0,0,0,0,46,46);
```

Comment:
This will make all stirge unagressive. Even the ones in other zones that should be, no?

Reply:
Yea - I guess that would actually. I spent like 20 mins looking through groups and such but I need to replace this and actually make it in mob_mods for group id 19 of pool id 3776.